### PR TITLE
update services with latest specs

### PR DESCRIPTION
**Pull request by cataggar**
I also added a CHANGELOG.md for the generated services.

I'm checking all the features in https://github.com/cataggar/azure-sdk-for-rust/actions/runs/1924895767 . If it runs for 6 hours , we are good.

### CODE REVIEW

File: services/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# 0.2 (2022-03)
+
+- generated from azure-rest-api-specs [commit from 2022-03-02](https://github.com/Azure/azure-rest-api-specs/commit/c56b44b794ff17caad4d31c6ab64b07e28253504)
```
Comment (cataggar) [inline services/CHANGELOG.md:3]: This is the commit used. It is now in the `CHANGELOG.md`.

File: services/mgmt/appplatform/Cargo.toml [hunk 2]
```diff
@@ -33,4 +33,3 @@ no-default-version = []
 "package-preview-2021-06" = []
 "package-preview-2020-11" = []
 "package-2020-07" = []
-"package-2019-05-01-preview" = []
```
Comment (heaths) [inline services/mgmt/appplatform/Cargo.toml:36]: Interesting to see a version removed. What prompted this?

**Review by heaths — APPROVED** [review]
Shared files and a sampling of generated files LGTM.

**Review by bmc-msft — COMMENTED** [review]
I thought we were going to bump the revs of all of the crates when regenerating with updated specs.

Even the crates with changes, I don't see version bumps.

File: services/mgmt/appplatform/Cargo.toml (hunk 2 above)
Comment (cataggar) [reply services/mgmt/appplatform/Cargo.toml:36]: The spec was removed. The easiest way to find the changes is to go to its readme and then look at history. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/appplatform/resource-manager/readme.md . That will take you to https://github.com/Azure/azure-rest-api-specs/pull/17506 where it was removed.

**Pull request merged**